go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ethereum/evmc/v7
```

Comment:
So it is enough to only declare it as "v7" here but use dir structure unchanged?

Reply:
yeah, it seems so